web/styles/subscriptions.css

```diff
@@ -266,6 +266,10 @@ h4.user_group_setting_subsection_title {
     padding-left: 5px;
 }
 
+.people-to-add-label {
```

Comment:
I realized you're riffing off an existing adjacent name, but I think `choose-subscribers-label` would be a lot better name for this.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/d8741f59de03a838e198d293d766333437997381..29af3356d5e69973fa588f17d436b08805086c47)